app/services/repository_rows/my_module_update_rows_service.rb

```diff
@@ -1,17 +1,24 @@
 # frozen_string_literal: true
 
 module RepositoryRows
-  class MyModuleAssigningService
+  class MyModuleUpdateRowsService
     extend Service
 
-    attr_reader :repository, :my_module, :user, :params, :assigned_rows_names, :errors
+    attr_reader :repository,
+                :my_module,
+                :user,
+                :params,
+                :assigned_rows_count,
+                :unassigned_rows_count,
+                :errors
 
     def initialize(my_module:, repository:, user:, params:)
       @my_module = my_module
       @repository = repository
       @user = user
       @params = params
-      @assigned_rows_names = Set[]
```

Comment:
So it does't return item names anymore?

Reply:
Before we used names for activities and flash messages. Now activities inside service. In flash messages we are showing only count of assigned/unassigned rows.